2022-03-19 21:21:44,968 INFO server.py Установлено соединение с клиентом: ('127.0.0.1', 52416)
2022-03-19 21:22:45,765 INFO server.py Получено сообщение от клиента {'action': 'presence', 'time': 1647699727.7356544, 'user': {'account_name': 'Guest'}}
2022-03-19 21:23:15,575 INFO server.py Сообщение от клиента прошло валидацию
2022-03-19 21:23:26,798 INFO server.py Отправлен ответ клиенту {'response': 200}
19-03-2022 21:35:59 INFO     server.py   Установлено соединение с клиентом: ('127.0.0.1', 52418)
19-03-2022 21:36:40 INFO     server.py   Получено сообщение от клиента {'time': 1647700500.7485693, 'user': {'account_name': 'Guest'}}
19-03-2022 21:36:49 ERROR    server.py   Сообщение от клиента {'time': 1647700500.7485693, 'user': {'account_name': 'Guest'}} не прошло валидацию
19-03-2022 21:54:35 INFO     server.py   Установлено соединение с клиентом: ('127.0.0.1', 52420)
19-03-2022 21:54:48 INFO     server.py   Получено сообщение от клиента {'action': 'presence', 'time': 1647701678.2073462, 'user': {'account_name': 'Guest'}}
19-03-2022 21:54:48 INFO     server.py   Сообщение от клиента прошло валидацию
19-03-2022 21:54:48 INFO     server.py   Отправлен ответ клиенту {'action': 200}
19-03-2022 22:01:12 INFO     server.py   Установлено соединение с клиентом: ('127.0.0.1', 52422)
19-03-2022 22:01:55 INFO     server.py   Получено сообщение от клиента {'action': 'presence', 'time': 1647702087.5817146, 'user': {'account_name': 'BGuest'}}
19-03-2022 22:02:01 ERROR    server.py   Сообщение от клиента {'action': 'presence', 'time': 1647702087.5817146, 'user': {'account_name': 'BGuest'}} не прошло валидацию
19-03-2022 22:02:14 INFO     server.py   Отправлен ответ клиенту {'respondefault_ip_addressse': 400, 'error': 'Bad Request'}

